c2corg_ui/templates/whatsnew.html

```diff
@@ -0,0 +1,42 @@
+<%inherit file="base.html"/>
+<%namespace file="helpers/common.html" import="show_title"/>
+
+<%block name="metarobots">
+  <meta name="robots" content="index,follow">
+</%block>
+
+<%block name="moduleConstantsValues">
+  ## Set in the data request
+  module.value('mapFeatureCollection', null);
+
+</%block>
+
+<section id="whatsnew">
+
+  <header>
+    <h1 class="text-center" translate>Feed of the latest changes</h1>
+  </header>
+
+  <div class="tools">
+
+##     <h3 class="heading show-phone" ng-click="mainCtrl.animateHeaderIcon($event)" data-toggle="collapse" data-target="#filters">
+##       <span translate>Filters</span><span class="glyphicon glyphicon-menu-down"></span>
```

Comment:
I don't think we need to have filters.
A link in the user's profiles towards ``whatsnew?u=12345`` should be enough.

Perhaps the client will need advanced filtering on changes in the future. But it's not required for now.